src/module/api/texupd/create/index.ts

```diff
@@ -20,6 +20,7 @@ export async function Create(newUpdate: INewUpdate): Promise<any> {
 
   objProperty.setText(newUpdate.text);
   obj.getMetadataMap().set(key.OrganizationID, newUpdate.organizationId);
+  obj.getMetadataMap().set(key.ventureID, newUpdate.ventureId);
```

Comment:
Is there a reason why this is now lowercase and all others are upper case?

Reply:
nope - just fixed consistency